src/core/config/remote/remote-config-validator.ts

```diff
@@ -35,12 +36,20 @@ export class RemoteConfigValidator implements RemoteConfigValidatorApi {
     this.localConfig = patchInject(localConfig, InjectTokens.LocalConfigRuntimeState, this.constructor.name);
   }
 
-  private static getRelayLabels(): string[] {
+  private static getRelayLabels(component: BaseStateSchema): string[] {
     // TODO:
     //  https://github.com/hashgraph/solo/issues/1823
     //  Add logic for selecting by specific label,
     //  when multiple instances can be deployed at the same time.
-    return [constants.SOLO_RELAY_LABEL];
+    const relayComponent = component as RelayNodeStateSchema;
+    const ids: number[] = relayComponent.consensusNodeIds;
+    console.log(`relayComponent.consensusNodeIds = ${JSON.stringify(relayComponent.consensusNodeIds)}`);
+    // generate label such as relay-node1 or relay-node1-node2
+    let label: string = 'relay';
+    for (const id of ids) {
+      label += '-' + Templates.renderNodeAliasFromNumber(id + 1);
```

Comment:
```suggestion
      label += `-${Templates.renderNodeAliasFromNumber(id + 1)}`;
```
consistency in how we join strings with variables and function return statements